Formula/zsh-navigation-tools.rb

```diff
@@ -17,7 +17,6 @@ def install
 
   def caveats; <<-EOS.undent
     To run zsh-navigation-tools, add the following at the end of your .zshrc:
-      fpath+=( #{HOMEBREW_PREFIX}/share/zsh-navigation-tools )
```

Comment:
Yes, I'm author of the software and from that version I've embedded fpath update within the sourced file